onnx/onnx-ml.proto

```diff
@@ -290,6 +290,12 @@ message GraphProto {
   // Initializers (see above) stored in sparse format.
   repeated SparseTensorProto sparse_initializer = 15;
 
+  // A list of named graphs, used to specify how model inputs should be 
+  // initialized. Each GraphProto must have a name (distinct from any other
+  // initializer) and one output, which is of the same type and shape as the
+  // corresponding model input.
+  repeated GraphProto graph_initializer = 16;
```

Comment:
Should we put the entire initialization stage into a single graph? I feel it's cleaner than having a list of graphs.